stl/src/filesystem.cpp

```diff
@@ -601,7 +596,7 @@ _Success_(return == __std_win_error::_Success) __std_win_error
             return {false, __std_win_error{GetLastError()}};
         }
         // check if FILE_ATTRIBUTE_READONLY is set
-        if (_Basic_info.FileAttributes & FILE_ATTRIBUTE_READONLY) {
+        if ((_Basic_info.FileAttributes & FILE_ATTRIBUTE_READONLY) != 0u) {

```

Comment:
I'm still curious why this is using `0u` instead of just `0`; no changes requested.